core/templates/components/entity-creation-services/exploration-creation.service.ts

```diff
@@ -68,7 +68,7 @@ export class ExplorationCreationService {
         );
         }, 150);
         return false;
-      }, function() {
+      }, () => {
```

Comment:
unwanted changes, remove them if they don't help this PR.

Reply:
I've encountered this before using `function` notation sometimes causes some delays resulting in weird e2e behaviour.